src/config/select.js

```diff
@@ -1,4 +1,3 @@
-import React from 'react';
```

Comment:
Check this: facebook.github.io/react/blog/2015/09/10/react-v0.14-rc1.html

React guys have still kept lot of old stuff in code(with warnings) which will finally go away after 0.15.